src/middleware/article.middleware.js

```diff
@@ -3,21 +3,24 @@ import Baserepository from '../repository/base.repository';
 import responseGenerator from '../helpers/responseGenerator';
 
 export const articleExist = async (req, res, next) => {
-  const { articleId } =
-    req.body.articleId !== undefined ? req.body : req.params;
-  const findArticle = await Baserepository.findOneByField(db.Article, {
-    id: articleId
-  });
+  const { articleId } = req.params || req.body;
```

Comment:
Good Job!👍

Reply:
Yeah, Thank you, and thanks to Princewill too.